packages/product-feed-google/src/Migrations/Version20230405111441.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Shopsys\ProductFeed\GoogleBundle\Migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Shopsys\MigrationBundle\Component\Doctrine\Migrations\AbstractMigration;
+
+class Version20230405111441 extends AbstractMigration
+{
+    /**
+     * @param \Doctrine\DBAL\Schema\Schema $schema
+     */
+    public function up(Schema $schema): void
+    {
+        $seoRobotsContentSettingCount = $this->sql(
```

Comment:
this variable name does not seem right

Reply:
Fixed. Too much inspired from other migrations. 😅